example/src/main/java/com/stripe/example/activity/FragmentExamplesFragment.kt

```diff
@@ -126,38 +127,18 @@ class FragmentExamplesFragment : Fragment() {
             return
         }
 
-        val isPaymentResult = stripe.onPaymentResult(
-            requestCode,
-            data,
-            callback = object : ApiResultCallback<PaymentIntentResult> {
-                override fun onSuccess(result: PaymentIntentResult) {
-                    viewModel.paymentIntentResultLiveData.value = Result.success(result)
-                }
-
-                override fun onError(e: Exception) {
-                    viewModel.paymentIntentResultLiveData.value = Result.failure(e)
+        if (stripe.isPaymentResult(requestCode, data)) {
```

Comment:
can you add a null check for data so we don't need to use `!!`?

Reply:
ditto